src/plugins/blocksmith/components/blocksmith/error-boundary/index.jsx

```diff
@@ -1,170 +1,99 @@
-import { Button } from '@triniti/admin-ui-plugin/components';
 import { connect } from 'react-redux';
-import { ContentBlock, EditorState } from 'draft-js';
-import blockParentNode from '@triniti/cms/plugins/blocksmith/utils/blockParentNode';
-import damUrl from '@triniti/cms/plugins/dam/utils/damUrl';
+import { EditorState } from 'draft-js';
 import PropTypes from 'prop-types';
 import React from 'react';
+import blockParentNode from '@triniti/cms/plugins/blocksmith/utils/blockParentNode';
+import createDelegateFactory from '@triniti/app/createDelegateFactory';
+import delegateFactory from '@triniti/cms/plugins/blocksmith/components/blocksmith/delegate';
```

Comment:
@hunyoboy should not use another component's delegate..why not just pass in handleStoreEditor as a prop?

Reply:
@efjacobson right i was just thinking that this component belongs under `blocksmith` folder so it could re-use the blocksmith's delegate. 
anyhow, since we will be passing the `handleStoreEditor` prop, we are gonna end up passing it from all the `Form` that's using the error-bounded blocksmith. Blocksmith,jsx is already using the delegate so we cannot re-use it. is that the approach you are thinking? just want to make sure.

Reply:
@hunyoboy ahh yes, that is weird. clearest solution is to just give error-boundary its own delegate